### PR TITLE
Add Makefile

### PR DESCRIPTION
**Pull request by lukaswajdzik**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+default: build
+
+.PHONY: clean
+clean:
+	@echo "Cleaning:"
+	mvn clean
```
Comment (krzysztofreczek) [inline Makefile:6]: Please use those commands in `.circleci/config.yml`

Comment (krzysztofreczek) [reply Makefile:6]: Please update README

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+.PHONY: build
+build:
+	@echo "Building maven artifacts:"
+	mvn clean install
+	@echo "...done"
+
+.PHONY: build-ci
+build-ci:
+	@echo "Building maven artifacts:"
```
Comment (krzysztofreczek) [inline Makefile:17]: Please add validation if all ENV variables are set

File: .circleci/config.yml [hunk 3]
```diff
@@ -17,7 +17,7 @@ jobs:
     <<: *defaults-env
```
Comment (krzysztofreczek) [inline .circleci/config.yml:17]: I believe README also requires some update when you describe how to run the code.

File: Makefile (hunk 2 above)
Comment (lukaswajdzik) [reply Makefile:19]: done

File: .circleci/config.yml (hunk 3 above)
Comment (lukaswajdzik) [reply .circleci/config.yml:17]: done, but not sure if all.. Please check

**Comment by krzysztofreczek** [comment]
🎖️

**Comment by krzysztofreczek** [comment]
@lukaswajdzik please remamber about squashing merges and removing branches after merge

**Pull request merged**